docs/report.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: page
+title: Report Security Issue
+toc: true
+---
+
+
+## I've found a vulnerability in Perl
+
+The Perl project takes security issues seriously. Please refer to the [Perl
+security report handling policy](https://perldoc.perl.org/perlsecpolicy) for a
+detailed description on how to report issues to the Perl security team. 
+
+The CPAN Security WG does not handle security issues in Perl, with the exception
+of dual-life core modules. We will coordinate with the Perl security team if
+necessary.
+
+
+## I've found malware on CPAN
+
+Please contact our security team on
+[cpan-security@perl.org](mailto:cpan-security@perl.org) privately, with the
+following information:
+
+- Distribution name, version, and package URL
+- Description of malicious behavior
+- Relevant logs, packet capture, code lines, screenshots or other context
+
+Examples of malware reports include: typo squatting, dependency confusion,
+exfiltration of private data, code obfuscation, outbound connections to
+malicious command and control servers, etc.
+
+## I've found a vulnerability in a distribution on CPAN
+
+If you believe you have found a security vulnerability in a distribution on
+CPAN, please follow the [Coordinated vulnerability
+disclosure](https://en.wikipedia.org/wiki/Coordinated_vulnerability_disclosure)
+model.
+
+It's important that you **do not** post information about it on:
+- bug trackers, like RT or GitHub
+- social media or chat channels, like IRC or Mastodon
+- mailing lists or discussion forums
+
+
+### Step 1: Prepare a Report
+
+Please provide a detailed description of the steps required to reproduce the
+vulnerability.
+
+The following information is useful:
```

Comment:
I think "Distribution name, version and package URL" is *required*, and not just "useful"...